digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala

```diff
@@ -537,7 +544,7 @@ class VVHClient(vvhRestApiEndPoint: String) {
       "HORIZONTALACCURACY",
       "VERTICALACCURACY",
       "VERTICALLEVEL",
-      "CONSTRUCTIONTYPE",
+      "CONSTRUCTIONTYPE",//TODO Remove this attribure from here because now is a instance variable
```

Comment:
It's better don't remove this attribute until the VVHHistoryRoadLink be modified like VVHRoadlink was.
We will modified the TODO for keep that information.